grakn-client/src/main/java/ai/grakn/client/Client.java

```diff
@@ -65,13 +64,13 @@
         }
     };
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) {
         int result;
         try {
             result = checkServerRunning();
         } catch (Exception e) {
-            LOG.error("An exception has occurred", e);
-            throw e;
+            LOG.error("An unexpected error occurred", e);
+            result = 2;
```

Comment:
WAT?

Reply:
This is the exit code. Here, we need to indicate "something went wrong" (so non-zero), but also differentiate it from the normal case of "engine is not running".
I think renaming it to `exitCode` should be enough to clear that up.

Reply:
UPDATE: I am using an enum like a good statically-typed programmer